packages/fast-components-react-base/src/label/label.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "../foundation";
+import {
+    ILabelHandledProps,
+    ILabelMangedClasses,
+    ILabelUnhandledProps,
+    LabelTag
+} from "./label.props";
+import { ILabelClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/* tslint:disable-next-line */
+class Label extends Foundation<ILabelHandledProps & IManagedClasses<ILabelClassNameContract>, React.HTMLAttributes<HTMLLabelElement>, {}> {
+    public static defaultProps: Partial<ILabelHandledProps> = {
+        tag: LabelTag.label
+    };
+
+    protected handledProps: HandledProps<ILabelHandledProps & IManagedClasses<ILabelClassNameContract>> = {
+        children: void 0,
+        managedClasses: void 0,
+        tag: void 0
+    };
+
+    /**
+     * Stores HTML tag for use in render
+     */
+    private get tag(): string {
+        return this.generateHTMLTag();
```

Comment:
You could simplify this by just returning `return LabelTag[this.props.tag]; ` from the getter - then you don't need the `generateHTMLTag` method